community-operators/service-binding-operator/0.0.20/service-binding-operator.v0.0.20.clusterserviceversion.yaml

```diff
@@ -18,7 +18,58 @@ spec:
       kind: ServiceBindingRequest
       name: servicebindingrequests.apps.openshift.io
       version: v1alpha1
-  description: Placeholder description
+  description: '
+  The Service Binding Operator enables users to create a managed binding between applications and operator-backed services such as databases, without having to perform manual configuration of secrets,  configmaps etc.
```

Comment:
I recommend replacing line 22 with this text:

The Service Binding Operator enables application developers to more easily bind applications together with operator managed backing services such as databases, without having to perform manual configuration of secrets, configmaps, etc. The Service Binding Operator accomplishes this through automatically collecting  binding information and sharing with an application to bind it with operator managed backing services. The binding is performed through a new custom resource called a “ServiceBindingRequest.”

Reply:
Done